src/frontend_workspaces/frontend/src/CugaHeader.css

```
@@ -87,17 +103,55 @@
+
+.cuga-user-menu-item:hover {
+  background-color: #e8e8e8;
+}
+
+.cuga-user-menu-item:active {
+  background-color: #e0e0e0;
+}
+
+.cuga-user-menu-item svg {
+  flex-shrink: 0;
+  fill: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)